### ISSUE TITLE
Stopped working in Chrome

### ISSUE THREAD

**Issue by marcelgood** [pmarks-net/ipvfoo#51]
With the recent Chrome updates the extension suddenly stopped working. It's just showing the IPvFoo logo instead of 6, 4 etc.

**Comment by pmarks-net** [comment]
Which version of Chrome?  It's working for me on 113.0.5672.46

**Comment by marcelgood** [comment]
Version 112.0.5615.138 (Official Build) (64-bit). That appears to be the latest stable version. I'm not on the dev channel.

**Comment by pmarks-net** [comment]
Does rebooting fix the problem?

If not, then right-click the IPvFoo icon, Manage extension, and toggle the switch Off/On.

**Comment by marcelgood** [comment]
Yes, it does. Actually even just logging out and then back in fixed it for now. I constantly have multiple Chrome windows open across multiple desktops, so it seems to stop working at some point.

**Comment by candlerb** [comment]
Thank you for this hint. Toggling it off an on again fixed it for me too (it had stopped working in Chrome 113, macOS)

**Comment by pmarks-net** [comment]
There is a Chrome bug (https://bugs.chromium.org/p/chromium/issues/detail?id=1316588) where IPvFoo stops working for about 3 minutes, then automatically resets itself.  I notice it happening (IPvFoo logo instead of 4/6) maybe a couple times a month.

So if you want to post a "me too", first wait 5 minutes and navigate to another page, to see if the 4/6 returns automatically.

I've never seen it get permanently stuck on my devices, but it would be nice to know if such a bug exists.

**Comment by npmstart-pray** [comment]
I've updated to Brave 1.58.124 (Chromium 117.0.5938.52) Linux 64bit and tried toggling and waiting, but no dice.
the foo is no goo...
